components/playground/playground.go

```diff
@@ -938,8 +1054,8 @@ func (p *Playground) bootCluster(ctx context.Context, env *environment.Environme
 
 	p.bootOptions = options
 
-	// All others components depend on the pd, we just ensure the pd count must be great than 0
-	if options.PDMode != "ms" && options.PD.Num < 1 {
```

Comment:
`dmmaster. num` defaults `0`?

Reply:
Good question，if we set dm as a default part for palyground maybe not a good idea，when user issue tiup playground，maybe this should not include start a dm cluster。 What do you think？

Reply:
> Good question，if we set dm as a default part for palyground maybe not a good idea，when user issue tiup playground，maybe this should not include start a dm cluster。 What do you think？@GMHDBJD @xhebox

Yes, I don't think it should be included here. And it did not break playground only because `PD.Num >= 1` in normal cases...

Reply:
dmmaster.num defaults 0, pd.num defaults 1, so tiup playground start a tidb cluster default, but I think we should allow user start a dm cluster without tidb/pd/tikv, so here change the logic to allow follow command.
```
tiup-playground nightly --dm-master 1 --dm-worker 1 --db 0 --kv 0 --pd 0 --tiflash 0 
```